source/1_engine/rex_engine/include/rex_engine/platform.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+namespace rex
+{
+#ifdef REX_PLATFORM_WINDOWS
```

Comment:
Do we want these conditional compilation statements or do we exclude the file in general from the build?